src/NUnitConsole/nunit3-console/Program.cs

```diff
@@ -86,6 +86,10 @@ public static int Main(string[] args)
                     return ConsoleRunner.OK;
                 }
 
+                // We already showed version as a part of the header
+                if (Options.ShowVersion)
```

Comment:
You're right, it's more user friendly than a blank screen. :-) 

Even though they told us to display it and not to display it - I know, we could generate a random number to decide! Just kidding, of course. :-)

I figured that if you asked for both version and help you wanted both - or maybe you want help about --version. In either case, you get what you want.